pkg/controller/backup_controller_test.go

```diff
@@ -340,6 +344,83 @@ func TestProcessBackupCompletions(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:           "backup with existing backup will fail",
```

Comment:
this test is not actually covering a scenario of an existing backup - see L349 and L364 - not sure if the name just needs to be updated to something like "backup without an existing backup in object storage will succeed", or if this is a dupe of an existing case?

Reply:
Great catch. It was a dupe name, just need to change that.